data/zilliqa/assets/zil1pxffll57y6u506m7njdm2dfnpjny0cmn2yz5pz/meta.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "Magusz Art",
+  "symbol": "MGZART",
+  "web": "https://maguszart.com/",
+  "links": {
+    "twitter": "magusz_art",
+    "web": "https://maguszart.com/"
```

Comment:
this property should be remove

Reply:
removed extra web link. 
and reduced file size <20kbs